### ISSUE TITLE
Create `woke init fuzz` command

### ISSUE THREAD

**Issue by michprev** [Ackee-Blockchain/woke#88]
`woke init fuzz` command will create the following directory structure:
- `pytypes` (the same functionality as `woke init types`)
- `tests`
    - `__init__.py` (empty)
    - `test_example.py`

Contents of `test_example.py`:
```python
import IPython
from woke import Campaign
from woke.fuzzer.decorators import flow, invariant, precondition, weight, ignore, max_times
from woke.fuzzer.random import random_string, random_account, random_int, random_bool, random_bytes


class TestingSequence:
    def __init__(self):
        pass
        # sequence setup code goes here
        # this setup is run at the beginning of each sequence

        # for example, contracts can be deployed here (in case they are not deployed in `test_example` function)

    @flow
    def flow_test_example(self):
        pass
        # flow code goes here
        # at least one transaction should be performed in this flow
        # assertions can be also made here

    @invariant
    def invariant_test_example(self):
        pass
        # invariant code goes here


# custom pytest fixtures are not supported
# but it is possible to use brownie's predefined fixtures (see https://eth-brownie.readthedocs.io/en/stable/tests-pytest-fixtures.html)
def test_example():
    # optional setup code goes here
    # any transactions performed here will be common to all testing sequences

    # typically, brownie contract objects will be passed to the `TestingSequence` constructor
    campaign = Campaign(lambda: TestingSequence())

    # the first argument specifies the number of sequences (i.e. the number of independent fuzzer runs)
    # the second argument specifies the number of flows applied in each sequence
    campaign.run(1000, 400)

```

**Comment by Aragahara** [comment]
.

**Comment by Aragahara** [comment]
Implemented in #109.

Examples are stored in examples/fuzzer directory and copied to the project directory/tests when `woke init fuzz` is used
